src/hooks/use-merge-data-dialog.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef, useState } from "react";
+import { useCustomModal } from "./use-custom-modal";
+import { ITileLinkMetadata } from "../models/tiles/tile-link-types";
+import { ITileModel } from "../models/tiles/tile-model";
+import { getTileModelById } from "../utilities/mst-utils";
+import MergeInIcon from "../../src/plugins/data-card/assets/merge-in-icon.svg";
+
+import "./link-tile-dialog.scss";
+
+interface IContentProps {
+  model: ITileModel;
+  selectValue: string;
+  hostTileTitle?: string;
+  mergableTiles: ITileLinkMetadata[];
+  setSelectValue: React.Dispatch<React.SetStateAction<string>>;
+}
+const Content: React.FC<IContentProps> = ({
+  model, selectValue, hostTileTitle, mergableTiles, setSelectValue
+}) => {
+  const selectElt = useRef<HTMLSelectElement>(null);
```

Comment:
Looks like this was copied over from other files but isn't being used in this dialog. You should remove it or duplicate how it's being used in the other files.

Reply:
It's being used as in the other three similar dialogs - it's there so that in the handler we can focus the select as soon as the button is clicked
 ```ts
setTimeout(() => selectElt.current?.focus())
```